demos/local-only-todolist/README.md

```diff
@@ -0,0 +1,74 @@
+# PowerSync + Supabase Flutter Local Only to Online Mode Demo: Todo List App
+
+This demo app is an extension of the Flutter Todo List App that demonstrates how to use the PowerSync SDK for Flutter in an local only way. It allows the user to use the app offline without having to sign in. After signing in data that was kept locally is synced up. For a step-by-step guide, see [here](https://docs.powersync.com/integration-guides/supabase).
```

Comment:
- of the *Supabase Todo List App
- in *a local only way

We could also make the intro sentence more explicit to help people grok the demo:

`This demo app is an extension of the Flutter Todo List App that demonstrates how to use the PowerSync SDK for Flutter in an local only way without sync capabilities.`